labcodes/labstorybook
# DSYS-18/alerts

Link to task:
https://labcodes.atlassian.net/browse/DSYS-18

How to test:
- With the LabSystem running, go to the "Component/System Messages" div in the LabStoryBook, then check if everything is rendered as expected.
- Locally, from labstorybook dir, run npm test, then check if all tests are passing.

This PR integrates `Alerts` and `Banners` to `master` branch after QA.
